test/jest/victory-tooltip/victory-tooltip.test.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Client tests
+ */
+/* global console */
+import React from "react";
+import { VictoryTooltip } from "victory-tooltip";
+import { fireEvent, render, screen } from "@testing-library/react";
+
+describe("components/victory-tooltip", () => {
+  const baseProps = {
+    x: 0,
+    y: 0,
+    datum: { some: "object" },
+    index: 3,
+    active: true,
+    text: "such text, wow"
+  };
+  beforeAll(() => {
+    jest.spyOn(console, "warn").mockImplementation(() => {});
+    jest.spyOn(console, "error").mockImplementation(() => {});
```

Comment:
I just merged in my PR, so you should be able to update this to use the `renderInSvg` function if you're still seeing these unrecognized DOM element errors.

Reply:
I switched to `renderInSvg`, and removed these spies.  However, I still do get this warning; any ideas how to fix it?
```
  console.warn
    `renderInPortal` is not supported outside of `VictoryContainer`. Component will be rendered in place

      63 |         if (typeof this.context.portalUpdate !== "function") {
      64 |           var msg = "`renderInPortal` is not supported outside of `VictoryContainer`. " + "Component will be rendered in place";
    > 65 |           Log.warn(msg);
         |               ^
      66 |           this.renderInPlace = true;
      67 |         }
      68 |
```